internal/pkg/describe/webapp.go

```diff
@@ -296,6 +336,28 @@ func (w *WebApp) HumanString() string {
 	for _, route := range w.Routes {
 		fmt.Fprintf(writer, "  %s\t%s\t%s\n", route.Environment, route.URL, route.Path)
 	}
+	fmt.Fprintf(writer, color.Bold.Sprint("\nVariables\n\n"))
+	writer.Flush()
+	fmt.Fprintf(writer, "  %s\t%s\t%s\n", "Name", "Environment", "Value")
+	var prevName string
+	var prevValue string
+	for _, variable := range w.Variables {
+		if variable.Name != prevName {
+			if variable.Value != prevValue {
+				fmt.Fprintf(writer, "  %s\t%s\t%s\n", variable.Name, variable.Environment, variable.Value)
+			} else {
+				fmt.Fprintf(writer, "  %s\t%s\t-\n", variable.Name, variable.Environment)
+			}
+		} else {
+			if variable.Value != prevValue {
+				fmt.Fprintf(writer, "  -\t%s\t%s\n", variable.Environment, variable.Value)
+			} else {
+				fmt.Fprintf(writer, "  -\t%s\t-\n", variable.Environment)
+			}
+		}
+		prevName = variable.Name
+		prevValue = variable.Value
+	}
```

Comment:
Can we extract this into a separate private method? `humanEnvVars() string`

Reply:
I don't think it needs to be a separate method. Hence making it a private method can reduce the length of the func, but this func has a consistent logic unless we do the same thing to each part (config env routes as well as resources). Also from implementation `humanString` has a buffer variable that holds all the data to be print, using the private method requires us to concatenate three pieces together. Does this make senses to you :P

Reply:
Not really important, we could have something like:
```go
var b bytes.Buffer
writer := tabwriter.NewWriter(&b, minCellWidth, tabWidth, cellPaddingWidth, paddingChar, noAdditionalFormatting)
fmt.Fprintf(writer, w.AboutSection())
fmt.Fprintf(writer, w.ConfigSection())
fmt.Fprintf(writer, w.RoutesSection())
fmt.Fprintf(writer, w.EnvVarsSection()) 
fmt.Fprintf(writer, w.ResourcesSection())
```
which is a bit easier to read but no big deal!

Reply:
Agree with both @sonofachamp and you. I think we can have a separate PR addressing this issue to make it another library. What do you think?